terraform/modules/paas/variables.tf

```
@@ -38,13 +38,15 @@ locals {
   redis_worker_service_name = "register-redis-worker-${local.app_name_suffix}"
   redis_cache_service_name  = "register-redis-cache-${local.app_name_suffix}"
   web_app_name              = "register-${local.app_name_suffix}"
-  app_environment           = merge(var.app_config_variable, var.app_secrets_variable)
-  review_app_start_command  = "bundle exec rake db:schema:load db:seed example_data:generate && bundle exec rails server -b 0.0.0.0"
-  web_app_start_command     = var.app_environment == "review" ? local.review_app_start_command : "bundle exec rails db:migrate && bundle exec rails server -b 0.0.0.0"
-  worker_app_start_command  = "bundle exec sidekiq -C config/sidekiq.yml"
-  worker_app_name           = "register-worker-${local.app_name_suffix}"
-  logging_service_name      = "register-logit-${local.app_name_suffix}"
-  web_app_routes            = [cloudfoundry_route.web_app_service_gov_uk_route.id, cloudfoundry_route.web_app_route.id]
+  app_environment = merge(var.app_config_variable, var.app_secrets_variable, {
+    BLAZER_DATABASE_URL = cloudfoundry_service_key.postgres-blazer-key.credentials.uri
```

This needs the settings prefix
```suggestion
    SETTINGS__BLAZER_DATABASE_URL = cloudfoundry_service_key.postgres-blazer-key.credentials.uri
```